File: lib/models/jwt_response.dart
class JwtResponse {
  final String token;
  final String type;
  final int id;
  final String email;
  final String name;

  JwtResponse({
    required this.token,
    required this.type,
    required this.id,
    required this.email,
    required this.name,
  });

  factory JwtResponse.fromJson(Map<String, dynamic> json) {
    return JwtResponse(
      token: json['token'],
      type: json['type'],
      id: json['id'],
      email: json['email'],
      name: json['name'],
    );
  }
}

File: backend/src/main/java/com/project/babyshophub/backend/payload/response/JwtResponse.java
package com.project.babyshophub.backend.payload.response;

public class JwtResponse {
    private String token;
    private String type = "Bearer";
    private int id;
    private String email;
    private String name;

    public JwtResponse(String accessToken, int id, String email, String name) {
        this.token = accessToken;
        this.id = id;
        this.email = email;
        this.name = name;
    }

    public String getAccessToken() {
        return token;
    }

    public void setAccessToken(String accessToken) {
        this.token = accessToken;
    }

    public String getTokenType() {
        return type;
    }

    public void setTokenType(String tokenType) {
        this.type = tokenType;
    }

    public int getId() {
        return id;
    }

    public void setId(int id) {
        this.id = id;
    }

    public String getEmail() {
        return email;
    }

    public void setEmail(String email) {
        this.email = email;
    }

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }
}